src/containers/Movie/Credits/component.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react'
+import {
+  Row, Col, Typography, Card
+} from 'antd'
+import PropTypes from 'prop-types'
+
+import { getImageUrl } from 'Utils'
+
+const Credits = ({ cast, crew, loading }) => (
+  !loading && (
+    <>
+      {Boolean(cast.length) && (
```

Comment:
looks like cast rows and crew rows are the same. lets create component for it

Reply:
@VladWithCookies layout for crew and cast are different